src/components/cylc/table/Table.vue

```diff
@@ -229,22 +229,22 @@ export default {
           sort: (a, b) => DEFAULT_COMPARATOR(a ?? '', b ?? '')
         },
         {
-          text: 'T-submit',
+          text: 'Submit',
           value: 'latestJob.node.submittedTime',
           sort: (a, b) => datetimeComparator(a ?? '', b ?? '')
         },
         {
-          text: 'T-start',
+          text: 'Start',
           value: 'latestJob.node.startedTime',
           sort: (a, b) => datetimeComparator(a ?? '', b ?? '')
         },
         {
-          text: 'T-finish',
+          text: 'Finish',
           value: 'latestJob.node.finishedTime',
           sort: (a, b) => datetimeComparator(a ?? '', b ?? '')
         },
         {
-          text: 'dT-mean',
+          text: '<Run-Time>',
```

Comment:
Not sure what the angled brackets are for
```suggestion
          text: 'Run Time',
```
or
```suggestion
          text: 'Mean Run Time',
```

Reply:
I was trying to avoid an unnecessarily wide column header, and angle brackets are common notation for "mean" or "expected", but I guess we can leave that implicit.